lib/pluginmanager/uninstall.rb

```diff
@@ -1,13 +1,16 @@
 # encoding: utf-8
```

Comment:
I just want to have something real quick for 5.0. I understand it is dup code, but I don't want to break stuff